NOTICE

```diff
@@ -1,4 +1,4 @@
-Apache Incubator-Heron
+Apache Heron
```

Comment:
is this expected?

Reply:
Sijie suggested we should make this change in the slack channel

Reply:
kk. thx.